IdentityCore/tests/MSIDInteractiveRequestParametersTests.m

```diff
@@ -145,6 +146,7 @@ - (void)testAllAuthorizeRequestParameters_whenOnlyAuthorizeParameters_shouldRetu
 - (void)testAllAuthorizeRequestParameters_whenOnlyTokenParameters_shouldReturnTokenParameters
 {
     MSIDInteractiveRequestParameters *parameters = [MSIDInteractiveRequestParameters new];
+    parameters.appRequestMetadata = nil;
```

Comment:
I think we should have tests when app request metadata is nil and non nil.

Reply:
Agree. Added tests for cases when app metadata is not nil.